### ISSUE TITLE
RTD builds failing due to excessive memory consumption in creating conda env

### ISSUE THREAD

**Issue by sgillies** [Toblerity/Shapely#858]
See https://readthedocs.org/projects/shapely/builds/10532048/

```
conda env create --quiet --name latest --file environment.yml
Collecting package metadata (repodata.json): ...working... Killed

Command killed due to excessive memory consumption
```

Looks like we need to prune in https://github.com/Toblerity/Shapely/blob/master/environment.yml.

**Comment by jorisvandenbossche** [comment]
It seems the last build was now successful after merging https://github.com/Toblerity/Shapely/pull/859: https://readthedocs.org/projects/shapely/builds/ 

In general, specifying the versions more helps reducing the memory usage of the solver.

**Comment by sgillies** [comment]
Resolved by #859. Yes, so many good reasons to pin the versions down.
